exporter/awsemfexporter/emf_exporter.go

```diff
@@ -33,6 +33,7 @@ import (
 	"go.uber.org/zap"
 
 	"github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/awsutil"
+	cloudwatch "github.com/open-telemetry/opentelemetry-collector-contrib/internal/aws/cwlogs"
```

Comment:
Let's remove the alias throughout the PR, so below things look like `cwlogs.Pusher` etc